### ISSUE TITLE
Potential conflict with Better Bibtex

### ISSUE THREAD

**Issue by jensedlund** [PubPeerFoundation/pubpeer_zotero_plugin#33]
When the Pubpeer plugin is loaded, the Better Bibtex plugin will get stuck loading on my system (Zotero 6 under Windows 11).

It seems likely that the reason is the same as described here: https://github.com/retorquere/zotero-better-bibtex/issues/1909 - the behaviour is identical.

Can replicate reliably by disabling/enabling the Pubpeer plugin.

**Comment by retorquere** [comment]
I've peeked into their code, and it looks like they're not loading the script that caused the breakage discussed in that issue.

**Comment by jensedlund** [comment]
No, I took the discussion to say that they removed that, but that it also needed to be removed from the other plugin mentioned for BB to loaf. So I guessed maybe Pubpeer was loadong that script or something similar, that interferes in the same way.

In any case, regardless of what causes this, BB loads fine if I disable Pubpeer, and freezes again if I enable it.  

But I've no experience with Zotero plugins at all, I just found that issue when I tried to find a fix for my own loading problem, and when I tried disabling plugins and found that disabling Pubpeer removes the issue, I thought I'd report.

**Comment by retorquere** [comment]
this has been fixed by pubpeer moving to restartless
